integration-tests/test_common_specs.py

```diff
@@ -26,6 +28,13 @@ def test_common_specs(insights_client, tmp_path):
         "insights.specs.Specs.uname.json",
         "insights.specs.Specs.yum_repos_d.json",
     ]
+    if platform.processor() == "x86_64":
+        common_specs.extend(
+            [
+                "insights.specs.Specs.dmidecode.json",
```

Comment:
Technically speaking, dmidecode is usable also on few more architectures other than x86_64; for example aarch64.